.travis.yml

```diff
@@ -1,14 +1,37 @@
+---
+os: osx
 sudo: true
-git:
-  depth: 10
+# lie to make travis work with nvm on osx
+language: cpp
+env:
+  matrix:
+    - TRAVIS_NODE_VERSION="0.10"
+    - TRAVIS_NODE_VERSION="0.12"
+    - TRAVIS_NODE_VERSION="2"
+    - TRAVIS_NODE_VERSION="4"
+    - TRAVIS_NODE_VERSION="5"
```

Comment:
Yeah I'm on OSX too and it again works flawlessly locally but Travis failed absolutely every single time without fail once consistent path got updated and similarly when switching to use consistent-env too it always reported the wrong path from npm get prefix